### DISCUSSION TITLE
install the hive after initial setup

### DISCUSSION THREAD

**Discussion by ecuthbert** [Security-Onion-Solutions/securityonion#2013] [Unsupported Versions] [answered]
Hello, I did a standalone installation without the hive module. 
Can I now, in a running system, install "the hive"?

Thanks in advance

**Comment by weslambert** [comment, marked as answer]
I can't recall if you are able to do it after the fact for TheHive, but if so, you could try changing the 0 to a 1 for thehive in `/opt/so/saltstack/local/minons/$minion.sls,` under the `manager` section, then run `so-thehive-start`.
